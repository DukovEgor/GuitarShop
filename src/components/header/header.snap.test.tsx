import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import Header from './header';

describe('Component: Header', () => {
  test('should render correctly', () => {
    const { container } = render(
      <BrowserRouter>
        <Header />
      </BrowserRouter>
    );

    expect(container).toMatchSnapshot();
  });
});
