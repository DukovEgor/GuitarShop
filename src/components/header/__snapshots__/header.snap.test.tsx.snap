// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Header should render correctly 1`] = `
<div>
  <header
    class="header"
    id="header"
  >
    <div
      class="container header__wrapper"
    >
      <a
        class="header__logo logo"
        href="/"
      >
        <img
          alt="Логотип"
          class="logo__img"
          height="70"
          src="/img/svg/logo.svg"
          width="70"
        />
      </a>
      <nav
        class="main-nav"
      >
        <ul
          class="main-nav__list"
        >
          <li>
            <a
              class="link main-nav__link link--current"
              href="/"
            >
              Каталог
            </a>
          </li>
          <li>
            <a
              class="link main-nav__link"
              href="/"
            >
              Где купить?
            </a>
          </li>
          <li>
            <a
              class="link main-nav__link"
              href="/"
            >
              О компании
            </a>
          </li>
        </ul>
      </nav>
      <div
        class="form-search"
      >
        <form
          class="form-search__form"
          id="form-search"
        >
          <button
            class="form-search__submit"
            type="submit"
          >
            <svg
              aria-hidden="true"
              class="form-search__icon"
              height="15"
              width="14"
            >
              <use
                href="#icon-search"
              />
            </svg>
            <span
              class="visually-hidden"
            >
              Начать поиск
            </span>
          </button>
          <input
            autocomplete="off"
            class="form-search__input"
            data-testid="search-input"
            id="search"
            placeholder="что вы ищите?"
            type="text"
          />
          <label
            class="visually-hidden"
            for="search"
          >
            Поиск
          </label>
        </form>
        <button
          class="form-search__reset"
          data-testid="reset-search"
          form="form-search"
          type="reset"
        >
          <svg
            aria-hidden="true"
            class="form-search__icon"
            height="15"
            width="14"
          >
            <use
              xlink:href="#icon-close"
            />
          </svg>
          <span
            class="visually-hidden"
          >
            Сбросить поиск
          </span>
        </button>
      </div>
      <a
        aria-label="Корзина"
        class="header__cart-link"
        href="/"
      >
        <svg
          aria-hidden="true"
          class="header__cart-icon"
          height="14"
          width="14"
        >
          <use
            xlink:href="#icon-basket"
          />
        </svg>
        <span
          class="visually-hidden"
        >
          Перейти в корзину
        </span>
        <span
          class="header__cart-count"
        >
          2
        </span>
      </a>
    </div>
  </header>
</div>
`;
